CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## [0.0.1] - 2022-08-16

```

Comment:
💡 Just thinking outloud: Is this the changelog versioning you guys are going with? Just thought is we want to keep it more basic, like: https://github.com/Azure/aks-periscope/blob/master/CHANGELOG.md#0010

Reply:
I was just following this - https://keepachangelog.com/en/1.0.0/, with the date actually it is nicer to see when the change happened, also with #Added, # Changed, #Removed is better. If you are saying this is standard, we follow for all our Azure repo's we can adhere to this https://github.com/Azure/aks-periscope/blob/master/CHANGELOG.md#0010 , no worries at all.

Reply:
I would rather to keep something with date on it. It makes live easier when debugging I think.  You can quickly see how old are things people are running with. As Hari mentioned if it is some Azure standard okay to change, but if not, I rather keep something with date :)

Reply:
☕️💡 cool, I guess my food for thought could be if we want to keep the current versioning strategy we should make it more visible/true to what we release  from change log version like for example : x.x.x.xxxxxx - dd.mm.yyhhmmss or whatever the versioning strategy this release will follow.

Reply:
So pipeline works here is screenshot 
<img width="941" alt="image" src="https://user-images.githubusercontent.com/105889062/185467781-fb7e9f95-a83c-4eb2-bb57-2814f4270e32.png">